docs/release-notes/allow-and-require-user-passwords.rst

```diff
@@ -0,0 +1,29 @@
+:orphan:
+
+**Breaking Changes**
+
+-  SDK and CLI: Enforce password requirements for all non-remote users matching what is required
+   when setting a password via the Web UI.
```

Comment:
-  Python SDK and CLI: Enforce password requirements for all non-remote users, aligning with WebUI password standards and having the following requirements: